@QiShi-NOAA Thanks for submitting this PR. First, I was able to reproduce the ops failure on Cactus by running EVS v1.0.22:

Log: /lfs/h2/emc/vpppg/noscrub/alicia.bentley/ops_fail/EVS/dev/drivers/scripts/stats/global_det/jevs_global_det_atmos_imd_grid2grid_stats_00.o224158657

```
BEGIN: global_det_atmos_stats_grid2grid_create_wind_shear.py
Traceback (most recent call last):
  File "/lfs/h2/emc/vpppg/noscrub/alicia.bentley/ops_fail/EVS/ush/global_det/global_det_atmos_stats_grid2grid_create_wind_shear.py", line 66, in <module>
    fhr = int(fhr_str)
ValueError: invalid literal for int() with base 10: ''
BEGIN: global_det_atmos_stats_grid2grid_create_anomaly.py
Using var_level = HGT_P500
Traceback (most recent call last):
  File "/lfs/h2/emc/vpppg/noscrub/alicia.bentley/ops_fail/EVS/ush/global_det/global_det_atmos_stats_grid2grid_create_anomaly.py", line 70, in <module>
    fhr = int(fhr_str)
ValueError: invalid literal for int() with base 10: ''
BEGIN: global_det_atmos_stats_grid2grid_create_wind_shear.py
```
I tried to test your fix, but it looks like noscrub filled up on Cactus. 

`mkdir: cannot create directory ‘old_logs’: Disk quota exceeded`

I'll test you fix tomorrow, but please make sure you don't need to make the same changes in:
global_det_atmos_stats_grid2grid_create_wind_shear.py

CC @sbanihash

--- a/ush/global_det/global_det_atmos_stats_grid2grid_create_anomaly.py
+++ b/ush/global_det/global_det_atmos_stats_grid2grid_create_anomaly.py
@@ -67,7 +67,7 @@ ENDDATE_dt = datetime.datetime.strptime(
 valid_date_dt = STARTDATE_dt
 while valid_date_dt <= ENDDATE_dt:
     for fhr_str in [f for f in fhr_list if f.strip()]:
-        fhr = int(fhr_str)        
+        fhr = int(fhr_str)
         init_date_dt = valid_date_dt - datetime.timedelta(hours=fhr)
         input_file = gda_util.format_filler(
             file_format, valid_date_dt, init_date_dt, str(fhr), {}

--- a/ush/global_det/global_det_atmos_stats_grid2grid_create_wind_shear.py
+++ b/ush/global_det/global_det_atmos_stats_grid2grid_create_wind_shear.py
@@ -62,7 +62,7 @@ ENDDATE_dt = datetime.datetime.strptime(
 )
 valid_date_dt = STARTDATE_dt
 while valid_date_dt <= ENDDATE_dt:
-    for fhr_str in fhr_list:
+    for fhr_str in [f for f in fhr_list if f.strip()]:
         fhr = int(fhr_str)
         init_date_dt = valid_date_dt - datetime.timedelta(hours=fhr)
         input_file = gda_util.format_filler(

